config/locales/models.de.yml

```
@@ -1107,6 +1108,7 @@ de:
       people: Lesen aller Personen, auf die du Zugriff hast
       invoices: Lesen aller Rechnungen, auf die du Zugriff hast
       mailing_lists: Lesen aller Abos, auf die du Zugriff hast
+      nextcloud: Lesen der eingestellten Nextcloud-Gruppen und des Anzeigenamens
```

```suggestion
      nextcloud: Lesen deiner Nextcloud-Gruppen und deines Anzeigenamens
```
Aus User-Sicht sind die Nextcloud-Gruppen keine Einstellungsmöglichkeit.